### PR TITLE
Add viewport meta tag

### PR DESCRIPTION
**Pull request by crhallberg**
Very useful when making sites look better when viewed on mobile. Much responsive. Wow.

Demo: http://www.crhallberg.com/beta/htmlshell/default.html
More info: https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-I often found myself whipping up some quick & simple html page in notepad(++) for whatever.
-
-Then I figured that being able to do something like open a new tab (because let’s face it, the browser is surely already open) and hit “htmlshell[CTRL+ENTER][CLICK][CTRL+C][ALT+TAB][CTRL+V]” was a lot easier (even if it doesn’t look that way at first glance ;) than typing out all those boilerplate tags yet again.
-
-Thus, here we are. If you find the site useful, awesome. If you feel like adding to it here, awesomer.
+I often found myself whipping up some quick & simple html page in notepad(++) for whatever.
+
+Then I figured that being able to do something like open a new tab (because let’s face it, the browser is surely already open) and hit “htmlshell[CTRL+ENTER][CLICK][CTRL+C][ALT+TAB][CTRL+V]” was a lot easier (even if it doesn’t look that way at first glance ;) than typing out all those boilerplate tags yet again.
+
+Thus, here we are. If you find the site useful, awesome. If you feel like adding to it here, awesomer.
```
Comment (crhallberg) [inline readme.txt]: I didn't change anything in this file, not sure what's up with that.

File: default.html [hunk 2]
```diff
@@ -115,16 +121,17 @@
     <code id="markup" onclick="selectCode()">
      <span>
       &lt;!DOCTYPE html&gt;<br>
-	 </span>
-	 <span id="htmlMT-div" style="display:block;">&lt;html&gt;<br></span>
-	 <span id="htmlIE-div" style="display:none;"></span>
-	 <span>
+     </span>
```
Comment (crhallberg) [inline default.html]: There were some tag characters mixed in with space indents. I replaced them with 4 spaces to make it all consistent.

**Comment by stefek99** [comment]
Thank you for PR.

I've neglecting this project (no changes) since I've took over domain.

As I merge another PR, now there are some conflicts...

> This branch has conflicts that must be resolved
> Use the command line to resolve conflicts before continuing.

...I'll get this done when I'm at home :)

**Comment by stefek99** [comment]
To busy with bit.ly/refugeespad

It's literally crazy, last few days putting some much effort into this event...

**Comment by crhallberg** [comment]
Should I open my PR on @raphaelz's version?

**Comment by stefek99** [comment]
I see that you've added new commit: https://github.com/crhallberg/htmlshell/commit/a3ef593639facf8901f9077cc0ce91ea1dffb1df

Opening PR on @raphaelz's? Would that change anything if site is served from here?

I actually cloned repo from @steveniles who then deleted the repo.

**Pull request merged**